### PR TITLE
BG2-2765: replace hello@ with a link to support page

### PR DESCRIPTION
**Pull request by dorota-joanna**

### CODE REVIEW

File: templates/password-reset-requested.html.twig [hunk 1]
```diff
@@ -4,7 +4,7 @@
   <p>Dear {{ firstName }},</p>
 
   <p>Please click on <a href="{{ resetLink }}">this link</a> to change your password. If you did not request a password
-    reset email, please contact us at hello@biggive.org for assistance.</p>
+    reset email, please visit the <a href="https://biggive.org/support/">Support</a> page for more help</p>
```
[the remark this answers is not in the document]
Comment (NoelLH) [reply templates/password-reset-requested.html.twig:7]: I think we should continue to use full sentences in line with the rest of the copy and other templates.

[the remark this answers is not in the document]
Comment (bdsl) [reply templates/password-reset-requested.html.twig:7]: I'm also not sure if linking to support here is very helpful - what we do we expect them to do after that? We know exactly what issue the issue is so its not like they should need to search for the issue. 

If we have a page what people should do who did not request a password we can link to that directly, if not doesn't it make more sense to link to https://biggive.org/contact-form/ if we want them to contact us?

[the remark this answers is not in the document]
Comment (bdsl) [reply templates/password-reset-requested.html.twig:7]: and as in other PR could we use small S for support? Big S with a definite article doesn't seem like correct English

**Comment by bdsl** [comment]
Agreed in standup not to change this.

**Pull request closed without merging**